app/Http/Controllers/API/v1/TransportController.php

```diff
@@ -408,6 +424,22 @@ public function create(Request $request): JsonResponse {
                 'additional'  => null, //unused old attribute (not removed so this isn't breaking)
             ];
 
+            // if isset, check in the other users with their default values
+            foreach ($withUsers ?? [] as $user) {
+                $checkin = TrainCheckinController::checkin(
```

Comment:
Needs to be refactored with the new hydrator

Reply:
Kannst du das bitte machen?